docs/doxygen/include/size_table.html

```diff
@@ -0,0 +1,20 @@
+<table>
+    <tr>
+        <td colspan="3"><center><b>Code Size of AWS IoT SigV4 Library (example generated with GCC for ARM Cortex-M)</b></center></td>
```

Comment:
Nit, this should be "AWS IoT SigV4" not "AWS IoT SigV4 Library" following our conventions from other libraries.

Reply:
Thanks, updated.